### PR TITLE
Testing Item Counter

### PR DESCRIPTION
**Pull request by Bateyjosue**
## Feature Description #3 

Make sure that the function for the items counter is covered with unit tests using Jest.

Cheer :tada: :tada: 
### Happy Review :clap: :clap: :clap:

### CODE REVIEW

File: src/__test__/data.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { countItem } from '../module/data.js';
+
+describe('ItemCounter', () => {
```
Comment (PraisesPJMT) [inline src/__test__/data.test.js:3]: [Optional] Consider adding a space between the test description to be "Meal Count"

File: src/__test__/data.test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  const array = [{ item_id: '001' }, { item_id: '002' }, { item_id: '003' }];
+  it('Should return the Exact length of the array of items', () => {
+    expect(countItem(array)).toBe(array.length);
+  });
+
+  it('[2, 5, 6, 5] Should return 4', () => {
+    expect(countItem([2, 5, 6, 5])).toEqual([2, 5, 6, 5].length);
+  });
```
Comment (PraisesPJMT) [inline src/__test__/data.test.js:11]: [Optional] Consider adding another test to check for returning zero count like
```
it('Should return zero for no-meal count', () => {
    expect(countItem([])).toBe(0);
  });
```

**Review by PraisesPJMT — COMMENTED** [review]
### **Hi @Bateyjosue,** 

### **Amazing Work! 🎉**

Your project is nice!  
I have some optional recommendations for you to consider! Remember they are all optional!
Also, don't forget to fix linter errors!
After that, then there is nothing else to say other than... it's time to merge it 🚀 : 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by shah-shuja-dev** [comment]
@Bateyjosue  
# Please Fix the linter error

**Review by shah-shuja-dev — APPROVED** [review]
## **Hello @Bateyjosue 👋🏻**

UJr

**Nice work implementing requested changes. Your tests passed with Flying Colors 💯** 
**There's nothing left to say other then it's time to merge it :shipit:**

# *STATUS :- APPROVED* 🟢
